### PR TITLE
Add Whitelist && update ReservedSlot extensions

### PR DESCRIPTION
**Pull request by ITeMbI4**
This PR adds Whitelist && update ReservedSlot extensions in Player.cs

### CODE REVIEW

**Review by VALERA771 — COMMENTED** [review]
Why we can't just use File.AppendAllLines() or smth else

**Review by louis1706 — CHANGES_REQUESTED** [review]
Don't make Breaking Change please keep the previous method and just add the new one with new args

**Review by VALERA771 — APPROVED** [review]

File: Exiled.API/Features/Player.cs [hunk 1]
```diff
@@ -1408,11 +1418,37 @@ public static Player Get(string args)
         /// <seealso cref="GiveReservedSlot()"/>
         public static bool AddReservedSlot(string userId) => ReservedSlot.Users.Add(userId);
 
+        /// <summary>
+        /// Adds a player's UserId to the list of reserved slots permanently. It will add a UserId to UserIDReservedSlots.txt file.
+        /// </summary>
+        /// <param name="userId">The UserId of the player to add.</param>
+        public static void PermanentlyAddReservedSlot(string userId) => ReservedSlots.Add(userId);
```
Comment (Thundermaker300) [inline Exiled.API/Features/Player.cs:1425]: Maybe instead of a different method, we can add a `bool permanent` to `AddReservedSlot`?

Comment (Thundermaker300) [reply Exiled.API/Features/Player.cs:1425]: Oh, that was the original intention. Thanks, Yamato

**Review by louis1706 — APPROVED** [review]

**Pull request merged**